rust/kcl-lib/src/execution/id_generator.rs

```diff
@@ -0,0 +1,63 @@
+//! A generator for ArtifactIds that can be stable across executions.
+
+use serde::{Deserialize, Serialize};
+
+use crate::execution::ModuleId;
+
+const NAMESPACE_KCL: uuid::Uuid = uuid::uuid!("29081a99-00fc-11f0-82ab-9cb6d03e6817");
+
+/// A generator for ArtifactIds that can be stable across executions.
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
```

Comment:
Pretty sure it doesn't need `Deserialize` or `Serialize` anymore.

Reply:
oh fair!